docs/UserGuide.md

```diff
@@ -48,6 +48,8 @@ HealthSync is a **powerful desktop application designed specifically for clinic
 > 3. Command Box
 > 4. Logger Tab
 
+>:bulb: Some outputs have to be scrolled to view.
```

Comment:
Should this be rephrased as `Longer outputs have to scrolled to be viewed`.

Reply:
Agreed, think this would be clearer.